src/model.rs

```diff
@@ -588,12 +588,13 @@ impl Collections {
             vj_id_to_old_idx.iter().map(|(id, idx)| (idx, id)).collect();
         comments_used.extend(self.stop_time_comments.iter().filter_map(
             |((old_vj_idx, _), old_comment_idx)| {
-                let old_vj_id = vj_idx_to_old_id[old_vj_idx];
```

Comment:
that line was failing because in `filter_ntfs` function we remove some vehicle journeys so `old_vj_idx` may not be contained in `vj_idx_to_old_id` anymore

Reply:
I just executed the binary to see if there are changes